dae/dae/genomic_resources/repository.py

```diff
@@ -738,21 +738,34 @@ def copy_resource(
             version_constraint=f"={remote_resource.get_version_str()}")
 
     def update_resource(

```

Comment:
It seems to me that this should be renamed to something like `update_resource_files`.

The behavior of this method when the set of files is `None` should be described in the doc string.